kuebiko/pipeline/nspl/flow.py

```diff
@@ -0,0 +1,215 @@
+from typing import Dict, Literal
+
+from metaflow import (
+    conda_base,
+    current,
+    FlowSpec,
+    Parameter,
+    project,
+    step,
+)
+
+
+GEOPORTAL_URL_PREFIX = "https://geoportal.statistics.gov.uk/datasets"
+
+# Type aliases
+postcode = str
+laua_name = str
+laua_code = str
+
+
+@conda_base(
+    ## In this flow we use Selenium to find the correct download link.
```

Comment:
@bishax Why are we using Selenium to find the link, rather than using the download link directly?